__tests__/nc_news.test.js

```diff
@@ -57,6 +57,43 @@ describe('GET /api/topics', () => {
 	});
 });
 
+describe('GET /api/users', () => {
+	describe('Endpoint Behaviour', () => {
+		test('GET:200 expects correct status code', () => {
+			return request(app).get('/api/users').expect(200);
+		});
+		test('GET:200 expects a copy of the user object', () => {
+			return request(app)
+				.get('/api/users')
+				.then(({ body }) => {
+					const users = body.users;
+					const userFormat = {
+						username: expect.any(String),
+						name: expect.any(String),
+						avatar_url: expect.any(String),
+					};
+					users.forEach((user) => {
```

Comment:
When you do a forEach you need to make sure you're checking the length, as they can produce false positives, so you need to make sure that something is actually being forEach'd :)